service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -67,9 +76,81 @@
     @Mock
     private GatekeeperEmailContentProvider gatekeeperEmailContentProvider;
 
+    @Mock
+    private C2UploadedEmailContentProvider c2UploadedEmailContentProvider;
+
+    @Mock
+    private IdamApi idamApi;
+
+    @Mock
+    private UserDetailsService userDetailsService;
+
     @InjectMocks
     private NotificationHandler notificationHandler;
 
+    @Test
+    void shouldNotNotifyHmctsAdminOnC2Upload() throws IOException, NotificationClientException {
+        final String SUBJ_LINE = "Lastname, SACCCCCCCC5676576567";
```

Comment:
what are your thoughts on using @Nested for these two tests here. Then setting SUBJ line and parameters outside the test

Reply:
we could use that actually